### DISCUSSION TITLE
So, we don't have SUPIR Upscaler/Resoration working in FORGE yet?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [lllyasviel/stable-diffusion-webui-forge#1667] [Q&A] [answered]
[SUPIR in Forge](https://github.com/Fanghua-Yu/SUPIR)

**Comment by barepixels** [comment]
Why does this need to be done in Forge?  Am asking because I believe Illyasviel'd time is valuable and limited.  Wouldn't it be better for him to focus on other important features/improvements/fixes.

**Comment by wzwow** [comment, marked as answer]
can put it in the forge SPACE

**Reply by ZeroCool22** [reply]
> can put it in the forge SPACE

That would be awesome!

But with the optimization/options for less VRAM usage.
